src/ol/source/vectortile.js

```diff
@@ -120,7 +120,7 @@ ol.source.VectorTile.prototype.getTile = function(z, x, y, pixelRatio, projectio
         tileCoord, projection);
     var tile = new ol.VectorImageTile(
         tileCoord,
-        urlTileCoord !== undefined ? ol.TileState.IDLE : ol.TileState.EMPTY,
+        urlTileCoord !== null ? ol.TileState.IDLE : ol.TileState.EMPTY,
```

Comment:
Do all falsey tile coords mean `EMPTY` or just `null`?

Reply:
`#getTileCoordForTileUrlFunction()` returns either an `ol.TileCoord` of `null`, but never `undefined` or other falsey values.